docs/style-guide.rst

```diff
@@ -233,7 +233,7 @@ Yes:
         bytes32[] options
     );
 
-    LongAndLotsOfArgs(
+    emit LongAndLotsOfArgs(
```

Comment:
Was `emit` not a thing when events were introduced, or is it still optional when emitting events?

Reply:
I think events initially used the call syntax just like functions. `emit` was introduced in 0.4.21 and was initially optional. Now it's mandatory.